app/test_main.py

```diff
@@ -1,3 +1,13 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zeros() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_default_return() -> None:
+    assert get_human_age(21, 15) == [1, 1]
+
+
+def test_age_28() -> None:
+    assert get_human_age(28, 28) == [3, 2]
```

Comment:
You have to use `pytest` to write tests. Besides that, make sure you cover all cases mentioned [in the checklist](https://github.com/Artemh59/py-cat-and-dog-years/blob/develop/checklist.md#clean-code) including error processing.